lib/saluki-components/src/sources/dogstatsd/mod.rs

```diff
@@ -791,6 +828,12 @@ fn handle_frame(
     let event = match parsed {
         ParsedPacket::Metric(metric_packet) => {
             let events_len = metric_packet.num_points;
+            for filter in filters.iter() {
+                if !filter.allow_metric(&metric_packet) {
+                    debug!(%metric_packet.metric_name, "Metric filtered out.");
+                    return Ok(None);
```

Comment:
Had a thought: what does the Agent do here in terms of telemetry? Should we tracking that we dropped this metric/packet?

Reply:
Doesn't look like there's any tracking https://github.com/DataDog/datadog-agent/blob/main/pkg/serializer/serializer.go#L336-L339